src/item_factory.cpp

```diff
@@ -545,8 +546,10 @@ void Item_factory::check_definitions() const
             }
         }
 
-        if( type->sym == 0 ) {
+        if( type->sym.empty() ) {
             msg << "symbol not defined" << "\n";
+        } else if( utf8_width( type->sym ) != 1 ) {
+            msg << "symbol must be exactly one console cell width" << "\n";
```

Comment:
Well, your suggestion would state something incorrect. One can have as many Unicode code points as one likes. Only together they must consume one character on the console. For example combination marks have a width of 0 and can be combined with other code points without increasing displayed length.